docs/hub/spaces-oauth.md

```diff
@@ -51,10 +51,10 @@ As for any other environment variable, you can use them in your code by using `o
 
 The allowed redirect URIs for your OAuth app are:
 
-- `https://{SPACE_SUBDOMAIN}.hf.space/auth/callback`
-- `https://{SPACE_SUBDOMAIN}.hf.space/login/callback`
+- `https://{SPACE_HOST}/auth/callback`
+- `https://{SPACE_HOST}/login/callback`
 
-Note that `SPACE_SUBDOMAIN` is also [available](https://huggingface.co/docs/hub/spaces-overview#helper-environment-variables) as an environment variable.
+Note that `SPACE_HOST` is also [available](https://huggingface.co/docs/hub/spaces-overview#helper-environment-variables) as an environment variable.
```

Comment:
Is this change backwards compatible?

Reply:
The variable is already in prod if it's the question

Reply:
yeah but is `SPACE_SUBDOMAIN` still exposed? to not break any existing Space that might use it

Reply:
(though it seems not to be used at least in public repos: https://huggingface.co/search/full-text?q=SPACE_SUBDOMAIN&type=space )

Reply:
Will remove it one month after this PR is merge if it's not used on any repo

Reply:
yep perfect